io-engine/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -1336,6 +1391,60 @@ pub async fn nexus_create_v2(
     }
 }
 
+async fn get_nexus_zone_info_from_children(children_devices: &mut Vec<(String, String)>) -> Result<ZoneInfo, Error> {
+    let mut zone_info = ZoneInfo::default();
+    let mut conventional = false;
+    let mut info_set = false;
```

Comment:
Is there another name for non-zoned blk devices btw? (rather than conventional)

```suggestion
    // if we find non-zoned block devices.
    let mut found_conventional = false;
    let mut zone_info: Option<ZoneInfo> = None;
```

Reply:
Usually, non-zoned blk devs are called "conventional". In the NVMe context, one could also label such a device as an NVM device (vs ZNS for zoned block devices), which refers to the I/O command set.